commerce-machine-learning-impl/src/main/java/com/liferay/commerce/machine/learning/internal/recommendation/data/source/UserCommerceMLRecommendationCPDataSourceImpl.java

```diff
@@ -93,6 +93,10 @@ public CPDataSourceResult getResult(
 			_commerceAccountHelper.getCurrentCommerceAccount(
 				httpServletRequest);
 
+		if (commerceAccount == null) {
+			return new CPDataSourceResult(new ArrayList<>(), 0);
```

Comment:
can we use `Collections.emptyList()` instead?

Reply:
done

Reply:
🙇